### DISCUSSION TITLE
config errors

### DISCUSSION THREAD

**Discussion by witsok** [Ysurac/openmptcprouter#2403] [General]
My setup is as follows:

RPI 4B
2 x LTE modems hooked up to 2 USB 3.0 ports in RPI
Amazon VPS
Laptop & RPI hooked up to a switch

RPI available at http://192.168.100.1/
Laptop card set up at 192.168.100.200

LTE modems have DHCP disabled and set up as
wan1 192.168.10.1 / IP 192.168.10.2 (multipath master, eth1, macVlan)
wan2 192.168.11.1 / IP 192.168.11.2 (multipath on, eth2, macVlan)
gateway for all 255.255.255.0

I can see the following errors

laptop - 'your IP was not leased by this router'

OMR 

- shadowsocks is not running
- VPN is not running

VPS server

- no server IP address (but it was added in the wizard)
- no public IP address
- can't contact server admin script (no token available)

LTE modems

- gateway down
- no server ping resonse after 1 second

What's more, I can't access any od the LTE modems.
I can see they log in (green light) but after a few minutes they get disconnected (green light starts blinking)

What do I do now?

**Comment by sidelong** [comment]
Did you find the solution to this? I am facing a very similar situation.

**Reply by iambenmitchell** [reply]
I've seen a similar issue and the solution is to disable interface renaming. I can't remember how to do it though, if you search for it on here you'll find it

**Reply by iambenmitchell** [reply]
https://github.com/Ysurac/openmptcprouter/issues/2524#issuecomment-1241547109
